Neos.Cache/Classes/Backend/MultiBackend.php

```diff
@@ -15,57 +15,48 @@
 
 use Neos\Cache\BackendInstantiationTrait;
 use Neos\Cache\EnvironmentConfiguration;
+use Neos\Flow\Core\Bootstrap;
+use Neos\Flow\Log\ThrowableStorageInterface;
+use Neos\Flow\Log\Utility\LogEnvironment;
+use Neos\Flow\ObjectManagement\ObjectManagerInterface;
+use Psr\Log\LoggerInterface;
+use Throwable;
```

Comment:
IMHO importing from the global namespace is confusing, but whatever. 😇

Reply:
I got used to it – otherwise PhpStorm subtly complains, so I obey …

Reply:
Oh, I disabled that for the global namespace… 😉